galsim/sed.py

```diff
@@ -146,12 +147,12 @@ def _initialize_spec(self):
         # Do some SED unit conversions to make internal representation proportional to photons/nm.
         # Note that w should have units of nm below.
         c = 2.99792458e17  # speed of light in nm/s
-        h = 6.62606957e-27 # Planck's constant in erg seconds
+        h = 6.62606957e-27  # Planck's constant in erg seconds
```

Comment:
These comments should be aligned.  If you (or flake8) feel the need to add a space here, add it before the `c` comment too.